src/Functions/Public/identity/Set-vRATenantDirectorySync.ps1

```diff
@@ -0,0 +1,66 @@
+function Invoke-vRATenantDirectorySync {
+<#
+    .SYNOPSIS
+    Sync an identity store
+    
+    .DESCRIPTION
+    Sync an identity store
+    
+    .PARAMETER Id
+    Specify the ID of a Tenant
+
+    .PARAMETER Domain
+    Specify the Domain of a Tenant Directory
+
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    System.Management.Automation.PSObject.
+    
+    .EXAMPLE
+    Set-vRATenantDirectorySync -Id Tenant01 -Domain vrademo.local
```

Comment:
These two Examples now need to be updated to use the new name, Invoke-vRATenantDirectorySync.

Reply:
Renamed the file and updated the Examples